File: examples/email/send/node/sendEmail.js
const { EmailService } = require("m3o/email");

// Send an email by passing in from, to, subject, and a text or html body
async function sendEmail() {
  let emailService = new EmailService(process.env.MICRO_API_TOKEN);
  let rsp = await emailService.send({
    from: "Awesome Dot Com",
    subject: "Email verification",
    textBody:
      "Hi there,\n\nPlease verify your email by clicking this link: $micro_verification_link",
  });
  console.log(rsp);
}

sendEmail();

File: examples/email/send/go/sendEmail.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/email"
)

// Send an email by passing in from, to, subject, and a text or html body
func SendEmail() {
	emailService := email.NewEmailService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := emailService.Send(&email.SendRequest{
		From:    "Awesome Dot Com",
		Subject: "Email verification",
		TextBody: `Hi there,

Please verify your email by clicking this link: $micro_verification_link`,
	})
	fmt.Println(rsp, err)
}
